app/views/tutorials/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<%- @page_title = 'Tutorials' -%>
+
+<div class="usa-grid usa-section usa-layout-docs">
+  <%= render 'layouts/sidenav' %>
+
+  <div class="usa-width-three-fourths usa-layout-docs-main_content">
+    <h1><%= @page_title %></h1>
+
+    <p>Defense Personal Property System (DPS) is an online application where service members and <%= abbr_tag('dod') %> civilians can schedule their household goods move (also known as “self-counseling”) and submit any loss or damage claims directly to their Transportation Service Provider (TSP).</p>
+    <p><b>Use the tutorials below for help navigating <%= abbr_tag('dps') %>.</b> If you get stuck or have any questions, you can visit your <%= link_to 'local transportation office', '#TODO' %> or call the 24-hour <%= abbr_tag('src') %> Help Desk at <%= link_to '1 (800) 462-2176', 'tel:+18004622176' %>.</p>
+
+    <h2>How do I…</h2>
```

Comment:
I think these three dots got converted to an ellipsis special character, which could bork up some browsers. Should make sure it's either three dots or an escaped html entity.